### PR TITLE
translated in french

### PR DESCRIPTION
**Pull request by JacquesHumule**

### CODE REVIEW

File: locale/fr/redmew.cfg [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+online=En ligne
+offline=Hors ligne
+player_index=Index des joueurs
+player_rank=Rang des joueurs
+yes=oui
+no=non
+time_played=Temps joué
+afk_time=Temps d'inactivité
+current_force=Current force
+current_surface=Current surface
```
Comment (JacquesHumule) [inline locale/fr/redmew.cfg:29]: Can i have the context ?

Comment (Cooldude2606) [reply locale/fr/redmew.cfg:29]: Think the context was removed, these keys are never used.
In fact only these three are used from common: "available" "quantity" "fail_no_target"

**Review by Cooldude2606 — APPROVED** [review]
All keys have been translated apart from current-force and current-surfae. I am happy to merge the changes once you have decided what you want to do with them.

**Comment by JacquesHumule** [comment]
I don't know how to translate it

**Comment by Cooldude2606** [comment]
No worries, since the keys are never used im happy to merge without a translation.

**Pull request merged**